homework/.ipynb_checkpoints/Homework1_alex_mrvaljevich-checkpoint.ipynb

```diff
@@ -0,0 +1,714 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# SYD DAT 3 Lab 1 - Git and Markdown\n",
+    "\n",
+    "##Homework:\n",
+    "\n",
+    "#### Setup\n",
+    "* Resolve any installation issues before next class.\n",
+    "* Make sure you have a github profile and created a repo called \"SYD_DAT_4\"\n",
+    "* Clone the class repo (this one!)\n",
+    "* Review this [code](../labs/Week 1/00_python_refresher.py) for a recap of some Python basics.\n",
+    "\n",
+    "#### Communication\n",
+    "* Read [Analyzing the Analyzers](http://cdn.oreillystatic.com/oreilly/radarreport/0636920029014/Analyzing_the_Analyzers.pdf) for a useful look at the different types of data scientists. Write down 5 key points you took away from the article\n",
+    "* Read about some [Markdown Techniques](http://daringfireball.net/projects/markdown/syntax)\n",
+    "* Write a summary of 2 chapters of [The Data Science Handbook](http://www.thedatasciencehandbook.com/) in Markdown and submit a pull request in the Lab Directory\n",
+    "\n",
+    "#### Programming\n",
+    "* Complete the lab from class and the additional Exercise below\n",
+    "\n",
+    "#### Course Project\n",
+    "* Come up with 5 different ideas for your course project. For each one list:\n",
+    "  * Overview of your idea\n",
+    "  * What data you will use\n",
+    "  * What the outcome is that you are trying to achieve\n",
+    "  * Any ideas of modelling techniques it may involve\n",
+    "\n",
+    "**Instructions: copy this file and append your name in the filename, e.g. Homework1_ian_hansel.ipynb.\n",
+    "Then commit this in your local repository, push it to your github account and create a pull request so I can see your work. Remeber if you get stuck to look at the slides going over Fork, Clone, Commit, Push and Pull request.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "\n",
+    "# Homework 1 - Analyzing the analyzers:\n",
+    "\n",
+    "1. Analyzing the Analizers 5 key point\n",
+    "\n",
+    "    * There is a lot of confusion about what a Data Scientist actually is, and it can cause a lot of trouble for both organizations and professionals\n",
+    "    * Clustering on self-reported personality traits is an awesome approach, keen to give it a twirl!\n",
+    "    * T-shaped skills are an interesting approach, i have to work on my statistics skills\n",
+    "    * As i mainly contract, i skew personally more towards Jack-of-all-trades, it's interesting to see it's representation in the data set.\n",
+    "    * As happened with Product Management 10 years ago, Data Science is a new role and we don't yet have a [Good Data Scientist / Bad Data Scientist](http://www.khoslaventures.com/wp-content/uploads/Good_Product_Manager_Bad_Product_Manager_KV.pdf) framework.  \n",
+    "    \n",
```

Comment:
I think a unifying theme behind all the varieties of data scientists is being comfortable with stats, so I'd strongly encourage you to get to a point where you are comfortable with that language.  I like that good product manager vs bad product manager link, I hadn't seen that before.